docs/config/README.md

```diff
@@ -1317,6 +1318,32 @@ The module will be shown if any of the following conditions are met:
 symbol = "⚙️ "
 ```
 
+## SHLVL
+
+The `shlvl` module shows the current SHLVL ("shell level") environment variable, if it is
+set and greater than the specified threshold.
```

Comment:
s/set/available? And maybe mention that this is supported in bash and zsh only:
https://www.gnu.org/software/bash/manual/html_node/Bash-Variables.html#index-SHLVL
http://zsh.sourceforge.net/Doc/Release/Parameters.html#Parameters-Set-By-The-Shell-1

Reply:
i didn't want to be too prescriptive incase that changes.